### ISSUE TITLE
Conda and pip crash with ../sysdeps/posix/getaddrinfo.c:2583: getaddrinfo: Assertion

### ISSUE THREAD

**Issue by Lynntee** [Microsoft/BashOnWindows#1331]
* A brief description
I just do nothing and then conda and pip crashed when I want to install or update some packages.
`apt-get` and `curl,wget` works well.
#614  doesn't work for me. 
* Expected results
It should begin to install packages.

* Actual results (with terminal output if applicable)
When I enter command `conda update conda`, terminal output is `Fetching package metadata ...python: ../sysdeps/posix/getaddrinfo.c:2583: getaddrinfo: Assertion '(__extension__ ({ const struct in6_addr *__a = (const struct in6_addr *) (sin6->sin6_addr.__in6_u.__u6_addr32); __a->__in6_u.__u6_addr32[0] == 0 && __a->__in6_u.__u6_addr32[1] == 0 && __a->__in6_u.__u6_addr32[2] == __bswap_32 (0xffff); }))' failed.
Aborted (core dumped)`

When I use pip, if the packages is already installed, it is ok, otherwise it crashes with the same output.

* Your Windows build number
10.1.14393
* Steps / All commands required to reproduce the error from a brand new installation 
`conda` or `pip` to install or update packages.

What can I do to fix it? Thanks.

**Comment by benhillis** [comment]
Thanks for opening this issue. When posting an issue the more information you can provide us the better job we can do to attempt to reproduce on our end and see what is wrong.  I installed conda from here and after installing I was able to run "conda update conda" successfully.

https://repo.continuum.io/miniconda/Miniconda2-latest-Linux-x86_64.sh

I am running build 14963 so it's possible this was fixed after the Anniversary update.

```
benhill@BENHILL-X1:~$ conda update conda
Fetching package metadata .......
Solving package specifications: ..........

# All requested packages already installed.
# packages in environment at /home/benhill/miniconda2:
#
conda                     4.2.12                   py27_0
```

**Comment by therealkenc** [comment]
This is same as #1202, #1113, #614 (and probably others). Was probably fixed in concert with #982.

**Comment by Lynntee** [comment]
@benhillis I'm sorry that I don't describe clearly. 

Actually I can use `conda` well before, but I just install some desktop like `unity` and  `xfce4,` then I remove them because they doesn't work, I'm now using `mate.` 

The problem is, I use `conda` to install a package ,and find it doesn't work, in fact I don't know when it crash, or whether it crash because of what I do before or not.

@therealkenc I check them out before, all of them don't work for me, thanks anyway.

Now `conda` and `pip` works!!! But I do nothing to fix them.

I don't know what happen. I find `pip` works, and `conda` still crashes last night, but today I try them again, I find that `conda` may crash but it works when I try it again. 

I try them many times before, they did crash everytime!

It's crazy. Thank you anyway.
